### PR TITLE
Update the set of mitigations, retune simulation

### PR DESCRIPTION
**Pull request by daeglin**
Finalni nebo temer finalni set opatreni. Jeste to neni playtestovano, kalibraci na CR budu resit pozdeji. Pridal jsem do game.ts seznam mitigaci, ktere se vzajemne vylucuji (funguji jako radio buttons)

### CODE REVIEW

**Comment by daeglin** [comment]
Uh, oh. Delam na tom zfailovanem lintu

File: frontend/src/app/services/game.ts [hunk 1]
```diff
@@ -58,59 +70,109 @@ export class Game {
     return this.simulation.getLastStats();
   }
 
-  get lastDate() {
+  get lastDate(): string {
```
Comment (jnwltr) [inline frontend/src/app/services/game.ts:73]: return type not needed for simple methods (just a clutter)

File: frontend/src/app/services/game.ts [hunk 2]
```diff
@@ -58,59 +70,109 @@ export class Game {
...
+    addMitigation('events100', 0.34, [0.12, 0.52], 20 * cs(), 10 * ss(), 'Akce 100 lidí');
+    addMitigation('events10', 0.42, [0.17, 0.60], 40 * cs(), 20 * ss(), 'Akce 10 lidí');
+    addMitigation('businessesSome', 0.18, [-0.08, 0.40], 20 * cs(), 15 * ss(), 'Rizikové služby');
+    addMitigation('businessesMost', 0.27, [-0.03, 0.49], 40 * cs(), 30 * ss(), 'Většina služeb');
+    addMitigation('universities', 0.19, [0.05, 0.33], 10 * cs(), 10 * ss(), 'Univerzity', {isSchool: true});
+    addMitigation('schools', 0.38, [0.16, 0.54], 50 * cs(), 40 * ss(), 'Všechny školy', {isSchool: true});
+    addMitigation('stayHome', 0.13, [-0.05, 0.31], 50 * cs(), 30 * ss(), 'Zůstat doma');
+    addMitigation('borders', 0.00, [0.00, 0.00], 10 * cs(), 10 * ss(), 'Zavřít hranice', {isBorders: true});
+
+/*
```
Comment (jnwltr) [inline frontend/src/app/services/game.ts:143]: please, remove

File: frontend/src/app/services/game.ts [hunk 3]
```diff
@@ -58,59 +70,109 @@ export class Game {
...
+    // isSchool mitigations are effective during school holidays "for free"
+    // isBorders controls epidemic drift across borders
+    function addMitigation(id: string, effectivity: number, effectivityConfidence: number[], costMPerDay: number, stabilityCost: number, label: string, flags?: any) {
       mitigations.push({
         id,
         label,
-        effectivity,
-        cost,
+        effectivity: effectivity + ens() * (effectivityConfidence[1] - effectivityConfidence[0]) / 4,
+        cost: costMPerDay,
```
Comment (jnwltr) [inline frontend/src/app/services/game.ts:172]: back and forth? let's minimize the changes, attributes do need explanations, you can document the method via jsdoc or the interface

**Review by jnwltr — CHANGES_REQUESTED** [review]

**Pull request closed without merging**